src/components/Button/Button.module.css

```diff
@@ -26,9 +26,12 @@
   width: 240px;
 }
 
-.authentication {
+.authentication, .joinGroup {
   background-color: var(--sec-blue-color);
   color: var(--white-color);
+}
+
+.authentication { 
   max-width: 270px;
```

Comment:
Just wanted to make sure that px were used intentionally

Reply:
Already changed to rems